modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,157 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService ingress controller
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller  overrides this, allowing for manual control.
+
+[NOTE]
+====
+This feature and procedure applies only to AWS. 
+====
+
```

Comment:
Is it necessary to have both this note and the prerequisite? I think the prerequisite is sufficient. In fact, when I first read this note I thought to myself, this should just be a prerequisite. Up to you though.